app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    if len(command.split()) == 3:
+        command_id, file_name, file_path = command.split()
+        if command_id != "mv":
+            raise ValueError("Command should be 'mv'")
+
+        if "/" not in command:
```

Comment:
Use `os.path.split`. "/" is a directory separator for Linux&MacOS, for Windows we use `\`